ydb/public/lib/ydb_cli/commands/ydb_service_scheme.cpp

```diff
@@ -543,6 +545,19 @@ int TCommandDescribe::DescribeReplication(const TDriver& driver) {
     return PrintDescription(this, OutputFormat, result, &TCommandDescribe::PrintReplicationResponsePretty);
 }
 
+int TCommandDescribe::PrintViewResponsePretty(const NYdb::NView::TDescribeViewResult& result) const {
+    Cout << "\nQuery text: " << result.GetViewDescription().GetQueryText() << Endl;
```

Comment:
I would add `\n` after the `Query text:` -- to make query text more separate and easy for the user to select.